pyaedt/edb_core/siwave.py

```diff
@@ -164,10 +183,12 @@ def impedance(self,value):
 
     @property
     def source_type(self):
+        """Property."""
```

Comment:
```suggestion
        """Source type."""
```
Is the description `Source type` more accurate?